cuebot/src/test/java/com/imageworks/spcue/test/service/JobManagerTests.java

```diff
@@ -454,7 +453,11 @@ public void optimizeLayer() {
         JobInterface job = getJob3();
         LayerDetail layer = layerDao.findLayerDetail(job, "pass_1");
 
-        assertEquals(Dispatcher.MEM_RESERVED_DEFAULT, layer.minimumMemory);
+        // Hardcoded value of dispatcher.memory.mem_reserved_default
+        // to avoid having to read opencue.properties on a test setting
```

Comment:
Same recommendation: Use cuebot/src/test/resources/opencue.properties for the value, rather than hardcoding it.